doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -157,14 +157,20 @@ compatible with the rest of |Coq|, up to a few discrepancies:
   (see :ref:`pattern_conditional_ssr`).  To use the
   generalized form, turn off the |SSR| Boolean ``if`` notation using the command:
   ``Close Scope boolean_if_scope``.
-+ The following two options can be unset to disable the incompatible
-  rewrite syntax and allow reserved identifiers to appear in scripts.
++ The following flags can be unset to make |SSR| more compatible with
+  parts of Coq:
 
-  .. coqtop:: in
+.. flag:: SsrRewrite
+
+   Controls whether the incompatible rewrite syntax is required.  The default
+   is "on".  "off" makes the syntax compatible with other parts of Coq.
```

Comment:
Let me advise once more against using this "on" / "off" terminology which might confuse readers in thinking that these would be options taking strings as input.

Reply:
On and off are the possible states of a switch. I don't find it confusing.

Reply:
At least the quotes are confusing. And we avoided this terminology for other entries because it *did* create some confusion for some readers.

Reply:
Yes, the `"` should not be there.

Reply:
I avoid using phrases such as "when the flag is set" because they are ambiguous.  It could mean both "when the flag has the value `on`" or "when the flag is assigned any value (both `on` or `off``).  `Test` and `Print Options` show the value of flags as `on` or `off`.  As I mentioned, I changed the wording to `enabled`.